mzLib/Omics/BioPolymer/IBioPolymer.cs

```diff
@@ -23,8 +24,8 @@ public interface IBioPolymer : IEquatable<IBioPolymer>
         /// second part is an alternative. The word "ORF", which stands for open reading frame in the first part
         /// is also acceptable. Here, the second part will include a range, for example "AO055_05240"
         /// </summary>
-        IEnumerable<Tuple<string, string>> GeneNames { get; }
-        IDictionary<int, List<Modification>> OneBasedPossibleLocalizedModifications { get; }
+        List<Tuple<string, string>> GeneNames { get; }
+        new IDictionary<int, List<Modification>> OneBasedPossibleLocalizedModifications { get; }
         char this[int zeroBasedIndex] => BaseSequence[zeroBasedIndex];
```

Comment:
How does the behaviour of BaseSequence and OneBasedPossibleLocalizedModifications change between IHasSequenceVariants and IBioPolymer?

Reply:
commented